doc/steal-topics/migrating-1.md

```diff
@@ -43,65 +53,78 @@ The following are breaking changes to the way StealJS works that will require yo
   main="app/index"></script>
 ```
+@highlight 2
 
-Where **app** is the `name` in your package.json (your package name). steal-tools now always writes bundles out to include the package name, so where it previously might have written to the file `dist/bundles/index.js` it now writes to `dist/bundles/app/index.js`.
+Where `app` is your package’s `name` in your `package.json`. `steal-tools` now always includes the package name in the bundles it produces, so where it previously might have written to the file `dist/bundles/index.js`, it now writes to `dist/bundles/app/index.js`.
 
-Using the new [configured steal.production](#easier-production-use) alleviates the need to know about this. The [StealJS.moving-to-prod] guide has been updated, and contains all of the ways to use StealJS in production.
+Using the new [configured steal.production](#easier-production-use) alleviates the need to know about this. The [StealJS.moving-to-prod] guide has been updated; it contains all of the ways to use StealJS in production.
 
 ### NPM 3
 
-The [npm] plugin now defaults to compatibility with npm 3 and above. If you're using Node version 5+, you have npm 3.  If you are using an older version of Node, or are using npm 2 for another reason, you need to update your configuration in your package.json to add:
+The [npm] plugin now defaults to compatibility with npm 3 and above. If you’re using Node version 5+, you have npm 3.  If you are using an older version of Node, or are using npm 2 for another reason, you need to update your configuration in your `package.json` to add:
 
 ```json
-"system": {
+"steal": {
   "npmAlgorithm": "nested"
 }
 ```
 
-If you *are* using npm 3+ and have `"npmAlgorithm": "flat"` in your package.json, you can remove this, but leaving it in your config will not cause any harm.
+If you *are* using npm 3+ and have `"npmAlgorithm": "flat"` in your `package.json`, you can remove this, but leaving it in your config will not cause any harm.
 
 ### CSS and Less plugins
 
-The CSS and Less plugins are no longer included with the steal npm package, but rather are contained within their own projects; **steal-css** and **steal-less**. To use them you need to install them and save them as devDependencies:
+The CSS and Less plugins are no longer included with the steal npm package, but rather are contained within their own projects: [steal-css](http://npmjs.com/package/steal-css) and [steal-less](http://npmjs.com/package/steal-less). To use them, install them as `devDependencies`:
 
 ```
-> npm install steal-css steal-less --save-dev
+npm install steal-css steal-less --save-dev
 ```
 
-And then update your package.json to use the new **plugins** configuration which tells Steal to load their package.json for metadata:
+Also update your `package.json` to use the new `plugins` configuration which tells Steal to load their `package.json` for metadata:
 
 ```json
-"system": {
+"steal": {
   ...
   
   "plugins": ["steal-less"]
 }
 ```
 
+### Stache plugin
```

Comment:
I don't want to include anything that makes it seem like StealJS is useful mostly in a DoneJS context.  I want StealJS to be generally useful as a module loader, and I'm afraid that constantly referring to CanJS makes it seem like it's just for CanJS projects.